sklearn/metrics/_classification.py

```diff
@@ -2680,7 +2689,14 @@ def log_loss(
 
     # Renormalize
     y_pred_sum = y_pred.sum(axis=1)
-    y_pred = y_pred / y_pred_sum[:, np.newaxis]
+    if (y_pred_sum != 1).any():
```

Comment:
This is too strict due to numerical precision. Something like
```python
if (!np.isclose(y_pred_sum, 1, rtol=1e-15, atol=5*eps).any():
```
might be enough.

Reply:
According to this condition a value of 1.1 seems to be acceptable. Should we still normalize then for such a value, to preserve the tests?